src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -84,35 +106,42 @@ public OSIsoftServer() {}
    * @param Headers Request headers
    * @param TextFields Request Payload
    * @param FileName File name for response
-   * @return The HTTP response code
+   * @return True/False message was successfully sent
    */
   public static boolean RequestHTTPS(
-      String CnxParam,
-      String Method,
-      String Headers,
-      String TextFields,
-      String FileName) {
-    int res = SCHttpUtility.HTTPX_CODE_NO_ERROR;
+      String CnxParam, String Method, String Headers, String TextFields, String FileName) {
+    int res = SCHttpUtility.HTTPX_CODE_EWON_ERROR;
+    String responseFileString = null;

```

Comment:
Please rename the variable to better indicate its purpose with the new changes. 

Suggestions:
1. `String responseBodyString`
2. `String responseString`
3. `String responseBodyContents`
4. etc...